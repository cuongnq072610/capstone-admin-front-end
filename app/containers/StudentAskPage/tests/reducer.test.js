import { fromJS } from 'immutable';
import studentAskPageReducer from '../reducer';

describe('studentAskPageReducer', () => {
  it('returns the initial state', () => {
    expect(studentAskPageReducer(undefined, {})).toEqual(fromJS({}));
  });
});
