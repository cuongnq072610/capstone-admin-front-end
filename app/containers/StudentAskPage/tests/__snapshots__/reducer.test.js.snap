// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`studentAskPageReducer returns the initial state 1`] = `
Immutable.Map {
  "isLoading": false,
  "asks": Immutable.List [],
  "errors": Immutable.Map {},
}
`;
